modal/cloud_bucket_mount.py

```diff
@@ -118,10 +118,8 @@ def f():
     read_only: bool = False
     requester_pays: bool = False
 
-    @property
-    def is_hydrated(self):
-        # Cloud bucket mounts do not have a server side representation.
-        return True
+    # Cloud bucket mounts do not have a server side object_id.
+    is_hydrated: ClassVar[bool] = True
```

Comment:
Would it make more sense to change the logic where we're reading this, e.g. to only check `if_hydrated` if the object is a subclass of `modal._Object`, or if it has an `is_hydrated` attribute or something?